pkg/must/must.go

```diff
@@ -0,0 +1,8 @@
+package must
+
+func Must[T any](obj T, err error) T {
```

Comment:
I feel this package to be a bit obscure, and I don't see why it should not be handled like any other error